HLTrigger/Configuration/python/HLTPhase2TDR_EventContent_cff.py

```diff
@@ -0,0 +1,68 @@
+import FWCore.ParameterSet.Config as cms
+
+HLTPhase2TDR = cms.PSet(
+    outputCommands = cms.vstring( *(
+        'keep *_TTTrackAssociatorFromPixelDigis_*_*',  
+        'keep *_TTStubAssociatorFromPixelDigis_*_*',  
+        'drop *_TTStubsFromPhase2TrackerDigis_*_HLT',
+        'drop Phase2TrackerDigiedmDetSetVectorPhase2TrackerDigiPhase2TrackerDigiedmrefhelperFindForDetSetVectoredmRefTTClusteredmNewDetSetVector_TTClustersFromPhase2TrackerDigis_ClusterInclusive_*',
+        'drop Phase2TrackerDigiedmDetSetVectorPhase2TrackerDigiPhase2TrackerDigiedmrefhelperFindForDetSetVectoredmRefTTClusterAssociationMap_TTClusterAssociatorFromPixelDigis_ClusterInclusive_*',
+        'drop Phase2TrackerDigiedmDetSetVectorPhase2TrackerDigiPhase2TrackerDigiedmrefhelperFindForDetSetVectoredmRefTTClusterAssociationMap_TTClusterAssociatorFromPixelDigis_ClusterAccepted_*',
+        'drop recoPFClusters_particleFlowClusterHGCal__*',
+        'drop l1tHGCalTriggerCellBXVector_hgcalVFEProducer_HGCalVFEProcessorSums_*',
+        'drop recoHGCalMultiClusters_ticlMultiClustersFromTrackstersMerge__*',
+        'drop recoHGCalMultiClusters_ticlMultiClustersFromTrackstersTrk__*',
+        'drop Phase2TrackerDigiedmDetSetVectorPhase2TrackerDigiPhase2TrackerDigiedmrefhelperFindForDetSetVectoredmRefTTClusteredmNewDetSetVector_TTClustersFromPhase2TrackerDigis_ClusterInclusive_HLT',
+        'drop recoHGCalMultiClusters_ticlMultiClustersFromTrackstersMIP__*',
+        'drop recoPFClusters_particleFlowClusterHGCalFromMultiCl__*',
+        'drop l1tHGCalClusterBXVector_hgcalBackEndLayer1Producer_HGCalBackendLayer1Processor2DClustering_*',
+        'drop Phase2TrackerDigiedmDetSetVectorPhase2TrackerDigiPhase2TrackerDigiedmrefhelperFindForDetSetVectoredmRefTTClusterAssociationMap_TTClusterAssociatorFromPixelDigis_ClusterInclusive_HLT',
+        'drop l1tHGCalTowerMapBXVector_hgcalTowerMapProducer_HGCalTowerMapProcessor_*',
+        'drop recoGsfTrackExtras_electronGsfTracks__*',
```

Comment:
This seems the wrong syntax in this and several other lines: the 4 fields are separated by single underscores, so __ should rather be _*_.

Reply:
thank you Martin, good spot! 

And you had me worried! However in this case it is unusual but correct. Its exactly matching the product which has no instance name rather than matching all instances. And it is dropping the product as expected. 

Its an artifact of how this was generated, from the spreadsheet I'll send you privately which has a very specific set of products we want to drop.  I do agree, we probably did want recoGsfTrackExtras_electronGsfTracks_*_*. Actually we probably should have done *_electronGsfTracks_*_*.  Will continue as is for now but if I need to update, I'll probably include those changes.